recipes/gslides/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "gslides" %}
+{% set version = "0.1.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/gslides-{{ version }}.tar.gz
+  sha256: 9d469e9bddaec14602aea35f97571db68d344615681ffb337cec20638b90db7f
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+  run:
+    - python >=3.7
+    - google-api-core >=1.26.2
+    - google-api-python-client >=2.0.2
+    - google-auth >=1.28.0
+    - google-auth-httplib2 >=0.1.0
+    - google-auth-oauthlib >=0.4.3
+    - googleapis-common-protos >=1.53.0
+    - ipython >=7.16.1
+    - numpy >=1.19.5
+    - pandas >=1.1.5    
+    - pyyaml >=5.4.1
+    - requests >=2.22.0
+    - tabulate >=0.8.9
+
+test:
+  imports:
+    - gslides
+    - tests
```

Comment:
This import succeeding means the package is installing a `tests` package in the global package namespace, which is no good. Something needs to be done to remove that, whether upstream or perhaps simply `rm -rf tests/` before running `pip install`.

Reply:
Quick question: could you please share a sample snippet as in where to insert `rm -rf tests/` (until it is fixed in the upstream)? I am curious. Thank you.

Is this what you meant?

```diff
 test:
   imports:
     - gslides
-    - tests
   commands:
+    - rm -rf tests/
     - pip check
   requires:
     - pip
```

Or, somewhere else?

Reply:
No it would need to be under `scripts`:
```yaml
script:
  - rm -rf tests/
  - {{ PYTHON }} -m pip install . -vv
```
This way `tests` won't be included in the built package.

Reply:
@dopplershift Thank you. Just applied it and all CI checks passed. Do, you think it is ready for merging now?